### ISSUE TITLE
Please provide a clear and actionable parameter directly.

### ISSUE THREAD

**Issue by lansinuote** [huggingface/trl#861]
These are sample commands in the documentation

```
torchrun --nnodes 1  --nproc_per_node 8 examples/stack_llama/scripts/supervised_finetuning.py --model_path=<LLAMA_MODEL_PATH> --streaming --no_gradient_checkpointing --learning_rate 1e-5 --max_steps 5000 --output_dir ./llama-se
torchrun --nnodes 1  --nproc_per_node 8 examples/stack_llama/scripts/reward_modeling.py --model_name=<LLAMA_SE_MODEL>
accelerate launch --multi_gpu --num_machines 1  --num_processes 8 examples/stack_llama/scripts/rl_training.py --log_with=wandb --model_name=<LLAMA_SE_MODEL> --reward_model_name=<LLAMA_SE_RM_MODEL> --adafactor=False --tokenizer_name=<LLAMA_TOKENIZER> --save_freq=100 --output_max_length=128 --batch_size=8 --gradient_accumulation_steps=8 --batched_gen=True --ppo_epochs=4 --seed=0 --learning_rate=1.4e-5 --early_stopping=True --output_dir=llama-se-rl-finetune-128-8-8-1.4e-5_adam
```

There are many placeholders in the command, such as "<LLAMA_ MODEL_ PATH>"

plz provide a clear and actionable parameter directly.

Otherwise, it's really difficult to debug these codes.

I don't know which parameters are expected by the program, and each program requires a lot of time to run, requiring repeated trial and error.

Especially the third command, thank you!

**Comment by younesbelkada** [comment]
hi @lansinuote , sorry for the confusion.
`LLAMA_SE_MODEL` stands for the path on the Hugging Face hub or local path of the llama model fine-tuned on stack-exchange, `LLAMA_SE_RM_MODEL` stands for the path on the Hugging Face hub or local path  for the Reward model fine-tuned on stack exchange and `LLAMA_TOKENIZER` stands for the path on the Hugging Face hub or local path or the llama tokenizer
